### PR TITLE
Feature/asset details graphql

### LINKED ISSUE #243: Fetch asset detail data using graphql

### PR DESCRIPTION
**Pull request by driels97**
This PR makes the app fetch the asset data from the backend using GraphQL just like the market data. With that, the blockchain websocket connection was removed since it is not necessary anymore, making the app load the asset data much faster.

Closes #243

### CODE REVIEW

File: lib/common/market_asset/data/repositories/asset_repository.dart [hunk 1]
```diff
@@ -11,25 +14,40 @@ class AssetRepository implements IAssetRepository {
...
+          }
+          listAssets.add(decodedAsset);
+        }
+      }
+
+      if (!fetchedPDEX) {
+        listAssets.add(
+          AssetModel.fromJson(
+            {
+              'asset_id': 'PDEX',
```
Comment (charleslukes) [inline lib/common/market_asset/data/repositories/asset_repository.dart:41]: make 'PDEX' a constant then share everywhere needed

File: test/common/market_asset/data/asset_repository_test.dart [hunk 2]
```diff
@@ -9,25 +10,39 @@ class _MockAssetRemoteDatasource extends Mock implements AssetRemoteDatasource {
...
           {
-            'assetId': '1',
-            'deposit': '18,000,000,000,000',
-            'name': 'TBTC',
-            'symbol': 'TBTC',
-            'decimals': '12',
-            'isFrozen': false
+          "asset_id": "PDEX",
+          "name": "PDEX",
+          "symbol": "POLKADEX",
```
Comment (charleslukes) [inline test/common/market_asset/data/asset_repository_test.dart:26]: make 'POLKADEX' a constant then share everywhere needed

File: test/common/market_asset/data/asset_repository_test.dart [hunk 3]
```diff
@@ -9,25 +10,39 @@ class _MockAssetRemoteDatasource extends Mock implements AssetRemoteDatasource {
...
+          "withdrawal_fee": "0"
           }
         ],
+        "nextToken": null
+      }
+    }''';
+
+    tGetAssetsEmpty = '''{
```
Comment (charleslukes) [inline test/common/market_asset/data/asset_repository_test.dart:34]: what does t in `tGetAssetsEmpty` mean?

File: lib/common/market_asset/data/repositories/asset_repository.dart (hunk 1 above)
Comment (driels97) [reply lib/common/market_asset/data/repositories/asset_repository.dart]: Solved

File: test/common/market_asset/data/asset_repository_test.dart (hunk 3 above)
Comment (driels97) [reply test/common/market_asset/data/asset_repository_test.dart:35]: It's just a variable created for unit test. This specific variable is used to test the case of a successfull request with no assets.

**Review by serhii-temchenko — APPROVED** [review]

**Pull request merged**